tests/testthat/test-onevax_xvwr.R

```diff
@@ -26,16 +26,17 @@ test_that("run_onevax_xvwr works correctly", {
     expect_equal(y2[[i]]$T[length(tt), , ], y3[[i]]$T[1, , ])
   }
 
-  uptake <- c(0.5, 1)
+  primary_uptake <- booster_uptake <- 0.5
   # check VoD is working correctly
   y3e <- run_onevax_xvwr(tt, gp, vea = 1, dur = 1e3, strategy = "VoD",
-                         uptake = uptake)
+                         primary_uptake = primary_uptake,
+                         booster_uptake = booster_uptake)
 
   for (i in seq_along(y3e)) {
     # no-one in stratum V is vaccinated again
     expect_equal(sum(y3e[[i]]$cum_vaccinated[, , 2]), 0)
     # uptake % of treated are vaccinated
-    expect_equal(y3e[[i]]$cum_vaccinated[, , -2] / uptake[i],
+    expect_equal(y3e[[i]]$cum_vaccinated[, , -2] / primary_uptake,
```

Comment:
```suggestion
    expect_equal(y3e[[i]]$cum_vaccinated[, , -2] / primary_uptake[i],
```
the rest of these will also need updating, sorry!